yarn.lock

```diff
@@ -11010,36 +11041,15 @@ isstream@~0.1.2:
   resolved "https://registry.yarnpkg.com/isstream/-/isstream-0.1.2.tgz#47e63f7af55afa6f92e1500e690eb8b8529c099a"
   integrity sha1-R+Y/evVa+m+S4VAOaQ64uFKcCZo=
 
-istanbul-api@^2.1.6:
-  version "2.1.6"
-  resolved "https://registry.yarnpkg.com/istanbul-api/-/istanbul-api-2.1.6.tgz#d61702a9d1c66ad89d92e66d401e16b0bda4a35f"
-  integrity sha512-x0Eicp6KsShG1k1rMgBAi/1GgY7kFGEBwQpw3PXGEmu+rBcBNhqU8g2DgY9mlepAsLPzrzrbqSgCGANnki4POA==
-  dependencies:
-    async "^2.6.2"
-    compare-versions "^3.4.0"
-    fileset "^2.0.3"
-    istanbul-lib-coverage "^2.0.5"
-    istanbul-lib-hook "^2.0.7"
-    istanbul-lib-instrument "^3.3.0"
-    istanbul-lib-report "^2.0.8"
-    istanbul-lib-source-maps "^3.0.6"
-    istanbul-reports "^2.2.4"
-    js-yaml "^3.13.1"
-    make-dir "^2.1.0"
-    minimatch "^3.0.4"
-    once "^1.4.0"
-
 istanbul-lib-coverage@^2.0.5:
```

Comment:
I'm not currently at a computer so I cannot see why this is still installed but it is concerning as 2.x is incompatible with 3.x.  I'll have a closer look once I can but wanted to mention this as soon as I saw it.

Reply:
Sorry I missed it in the first look, karma-esm also depends on babel-plugin-istanbul@5.  This needs to be upgraded to v6.

Reply:
Upgrading babel-plugin-istanbul to 6.0.0 in the karma-esm package works, but `@storybook/addon-docs` depends on `@jest/transform` which relies on `babel-plugin-istanbul@^5.1.0` and it looks like storybook addon docs hasn't been updated yet. I'm unsure how to proceed here.

Reply:
I'll see if we can lose this dependency

Reply:
https://github.com/open-wc/open-wc/pull/1590 removes the dependency